libphal/phal_dump.C

```diff
@@ -287,9 +287,12 @@ void writeDumpFile(char* data, size_t len, std::filesystem::path& dumpPath)
 void collectSBEDump(uint32_t id, uint32_t failingUnit,
 		    const std::filesystem::path& dumpPath)
 {
+	constexpr uint16_t ODYSSEY_SBE_DUMP = 11;
```

Comment:
for now just check whether input is proc or not, if not return an error

Reply:
Changed

Reply:
I am not seeing any change here

Reply:
It is at LN297 we are returning error just after logging the entry log